rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/rm/ListNodeCommand.java

```diff
@@ -53,7 +53,7 @@ public ListNodeCommand(String nodeSource) {
 
     @Override
     public void execute(ApplicationContext currentContext) throws CLIException {
-        HttpGet request = new HttpGet(currentContext.getResourceUrl("monitoring"));
+        HttpGet request = new HttpGet(currentContext.getResourceUrl("monitoring/full"));
```

Comment:
I would call it `/all` instead of `/full`.

Reply:
Fabien is for /full

Reply:
Ok for me.